### PR TITLE
Slack notifications with slack notify

### PR DESCRIPTION
**Pull request by mauro-oto**
Hey @etagwerker, @alanhala, 

This PR adds a new post to the blog, regarding Slack notifications using `slack-notify`. 

Please check it out, any suggestions or comments are welcome. Thanks!

### CODE REVIEW

File: _posts/2015-07-29-slack-notifications-with-slack-notify.markdown [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Slack notifications with Slack-Notify gem"
+date: 2015-07-29 17:40:00
+categories: ["ruby"]
+author: "mauro-oto"
+---
+
+We have been using Slack at Ombulabs for a while now after switching from
```
Comment (mauro-oto) [inline _posts/2015-07-29-slack-notifications-with-slack-notify.markdown]: OmbuLabs should have a link to ombulabs.com and uppercase L

File: _posts/2015-07-29-slack-notifications-with-slack-notify.markdown [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+HipChat, and haven't looked back. It looks and feels much better than any other
+available platform of its kind. Slack provides
+[WebHooks](https://api.slack.com/incoming-webhooks), which you can use to post
+messages to your team's channels.
+
+We use [Solano CI](https://www.solanolabs.com) (formerly Tddium) for our
+automated builds, and we would get e-mails whenever a build passed or failed,
```
Comment (mauro-oto) [inline _posts/2015-07-29-slack-notifications-with-slack-notify.markdown]: Separate in two sentences, `automated builds. By default, the system... `

File: _posts/2015-07-29-slack-notifications-with-slack-notify.markdown [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+but we wanted to be notified in our Slack channels. Enter the
+[slack-notify](https://github.com/sosedoff/slack-notify) gem, which makes
+notifying Slack using Ruby a breeze.
```
Comment (mauro-oto) [inline _posts/2015-07-29-slack-notifications-with-slack-notify.markdown]: which makes Slack notifications super simple in Ruby.

File: _posts/2015-07-29-slack-notifications-with-slack-notify.markdown [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+To get started, first
+[set up an incoming webhook](https://my.slack.com/services/new/incoming-webhook).
```
Comment (mauro-oto) [inline _posts/2015-07-29-slack-notifications-with-slack-notify.markdown]: Do we need to enter any info here?

File: _posts/2015-07-29-slack-notifications-with-slack-notify.markdown [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    notify_slack(msg)
+  end
+end
+
+before "deploy", "notify:start"
+after "deploy", "notify:done"
+```
+
+There are many different use cases, just make sure not to spam your team with
+too many notifications. At some point, you will probably want to set up a
```
Comment (mauro-oto) [inline _posts/2015-07-29-slack-notifications-with-slack-notify.markdown]: "you may want to"

File: _posts/2015-07-29-slack-notifications-with-slack-notify.markdown [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  end
+end
+
+before "deploy", "notify:start"
+after "deploy", "notify:done"
+```
+
+There are many different use cases, just make sure not to spam your team with
+too many notifications. At some point, you may want to set up a
+channel dedicated solely to notifications if you are a bigger team.
```
Comment (etagwerker) [inline _posts/2015-07-29-slack-notifications-with-slack-notify.markdown]: @mauro-oto I'd say "if you are part of a bigger team."

**Pull request merged**